### PR TITLE
Add menu options to change the zoom level on the webkit component

### PR DESCRIPTION
**Pull request by andrecasteliano**
Creates two new menu options (Zoom In and Zoom Out) to control the zoom level on the webkitview component.

### CODE REVIEW

File: src/WebView.cpp [hunk 1]
```diff
@@ -130,3 +130,21 @@ void WebView::refresh()
 {
     webkit_web_view_reload(*this);
 }
+
+void WebView::zoomIn()
+{
+    double zoomLevel { webkit_web_view_get_zoom_level(*this) };
```
Comment (eneshecan) [inline src/WebView.cpp:136]: Please use `auto zoomLevel = ...`

File: src/WebView.cpp [hunk 2]
```diff
@@ -130,3 +130,21 @@ void WebView::refresh()
...
+    if (zoomLevel >= 2) return;
+
+    zoomLevel += 0.05;
+    webkit_web_view_set_zoom_level(*this, zoomLevel);
+}
+
+void WebView::zoomOut()
+{
+    double zoomLevel { webkit_web_view_get_zoom_level(*this) };
```
Comment (eneshecan) [inline src/WebView.cpp:145]: Please use `auto zoomLevel = ...`

File: src/WebView.cpp [hunk 3]
```diff
@@ -130,3 +130,21 @@ void WebView::refresh()
...
+    if (zoomLevel <= 0.5) return;
```
Comment (eneshecan) [inline src/WebView.cpp:146]: Please use brackets on the next line for convention consistency.

**Review by eneshecan — CHANGES_REQUESTED** [review]

**Comment by eneshecan** [comment]
Thanks a lot for this! This definitely looks worse but we get to keep the accelerators and imho fanciness about zoom is not a big deal.

**Comment by eneshecan** [comment]
Just remembered something; it's better we keep this in `Settings` but I can do that after merging.

File: resource/ui/MainWindow.ui [hunk 4]
```diff
@@ -14,6 +14,24 @@
...
+    </child>
+    <child>
+      <object class="GtkMenuItem" id="zoomout_menu_item">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="label" translatable="yes">Zoom Out</property>
+        <property name="use_underline">True</property>
+        <accelerator key="minus" signal="activate" modifiers="GDK_CONTROL_MASK"/>
+      </object>
+    </child>
```
[the remark this answers is not in the document]
Comment (andrecasteliano) [reply resource/ui/MainWindow.ui:49]: I created a submenu for the zoom options

File: src/WebView.cpp [hunk 5]
```diff
@@ -130,3 +130,21 @@ void WebView::refresh()
 {
     webkit_web_view_reload(*this);
 }
+
+void WebView::zoomIn()
+{
+    double zoomLevel { webkit_web_view_get_zoom_level(*this) };
+    if (zoomLevel >= 2) return;
```
[the remark this answers is not in the document]
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp (hunk 1 above)
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp (hunk 2 above)
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp (hunk 3 above)
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

**Comment by andrecasteliano** [comment]
New settings key 'zoom_level' was added. Take a look later and let me know if it's ok.

**Comment by eneshecan** [comment]
Looks great! I will merge when I have some time.

**Pull request merged**